rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -89,14 +90,20 @@
     </build>
 
     <dependencies>
-        <!-- MP LRA CDI support relies on JAX-RS filters for
-             starting LRAs and for enlisting resources in LRAs
-        -->
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-narayana-lra</artifactId>
```

Comment:
please double check with the Narayana team if this is really what you want to do in the quickstart since Quarkus extension for Narayana LRA has a hardcoded (different) version of narayana than the current master branch.

Reply:
Thanks for the review, I confirm it does work when I replace quarkus-narayana-lra with narayana-lra + lra-service-base. Otherwise I get 503, service unavailable.

Reply:
full log is: 
``` Exception start LRA client request timed out, try again later': container request for method 'io.narayana.rts.lra.CdiBasedResource#doInTransaction': lra: 'context'
2022-07-22 10:33:49,571 WARN  [io.nar.lra] (executor-thread-0) LRA025025: Unable to process LRA annotations: -3: StartFailed (start LRA client request timed out, try again later) ()'
  0   119    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
HTTP/1.1 503 Service Unavailable ```